src/calendar/DatePicker.tsx

```diff
@@ -58,10 +59,11 @@ export interface DatePickerProperties extends ThemedProperties {
 	onRequestYearChange?(year: number): void;
 	renderMonthLabel?(month: number, year: number): string;
 	year: number;
+	yearBasis?: number;
```

Comment:
How about naming this `initialYear`?

Reply:
The `yearBasis` is used as an offset to inform the year drop-down how to number the pages. Maybe `yearPageOffset` or `yearNumberingOffset` or `initialYearOffset` and set the default value to 0?